src/header.tsx

```diff
@@ -62,7 +62,7 @@ export type HeaderMenuProps = {
 export const HeaderMenu: React.FC<HeaderMenuProps> = ({ close, items, label, breakpoint }) => {
   const config = useAppkitConfig();
   const isDark = useColorScheme().scheme === "dark";
-  const bgColor = isDark ? config.colors.neutral15 : config.colors.neutral05;
+  const bgColor = useColorScheme().scheme ? config.colors.neutral15 : config.colors.neutral05;
```

Comment:
Uhm what does this do? `scheme` is always a non-empty string so this will always be neutral15 right? 

But yes the `isDark` is kind of wrong now.

Reply:
Another copy & paste error, reverting for now.